sdk/simulator-service/src/strategies/refinance/RefinanceLendingToLending.ts

```diff
@@ -148,5 +191,66 @@ export async function refinanceLendingToLending(
     targetPosition,
     swaps: Object.values(simulation.swaps),
     steps: Object.values(simulation.steps),
-  } as ISimulation<SimulationType.Refinance>
+  } satisfies ISimulation<SimulationType.Refinance>
+}
+
+/**
+ * CalculateBorrowAmount
+ * @description Determines how much to borrow.
+ *    When the DebtSwap step is skipped we simply return the previous position's debt amount
+ *    When a DebtSwap is required we need to borrow enough to cover the original flashloan after
+ *    accounting the swap and assuming the worst case scenario on slippage IE max slippage.
+ *
+ *    We also need to factor in Summer fees ahead of time
+ */
+async function calculateBorrowAmount(params: {
```

Comment:
Perhaps move to a utils file? Only if you think it makes sense

Reply:
Might leave it bcause i think it'll be quite specific to this use case. Might be wrong